com.unity.ml-agents/CHANGELOG.md

```diff
@@ -10,11 +10,10 @@ and this project adheres to
 ## [Unreleased]
 ### Major Changes
 #### com.unity.ml-agents (C#)
+- Agent with both continuous and discrete actions is now supported. You can specify
```

Comment:
nit: This is still a minor change from semantic versioning point of view. We'll list it as "major" for the release notes though.

Also, please keep the old PR# (#4702) and add this one.

Reply:
was following the python changelog. Is the python changes also minor?